VCProjectEngineWrapper/VCCLCompilerToolWrapper.cs

```diff
@@ -79,17 +81,17 @@ public string GetToolPath()
 
 		public string[] GetAdditionalIncludeDirectories()
 		{
-			return _wrapped.AdditionalIncludeDirectories.Split(';');
+			return _wrappedRules.GetEvaluatedPropertyValue("AdditionalIncludeDirectories").Split(';');
```

Comment:
Main change: access property values through `IVCRulePropertyStorage` wherever we can.